integration-test/scripts/fetch-secrets.sh

```diff
@@ -14,7 +14,7 @@ fi
 mkdir -p ./src/main/resources/ums-users
 
 echo "Executing secret fetching from Azure 'jenkins-secret' store"
-az keyvault secret show --name "real-ums-users-dev" --vault-name "jenkins-secret" --query 'value' -o tsv | jq > $USER_JSON_LOCATION
```

Comment:
why create a new map? 
`az keyvault secret show --name "real-ums-users-dev" --vault-name "jenkins-secret" --version $secret_version --query 'value' -o tsv | jq > $USER_JSON_LOCATION`

I've a similar approach in another PR. This way we can easily update the secret.

Reply:
sure, makes sense, i'll wait for your pr and rebase onto that commit to use that solution